lua/entities/kleiner/sv_spawning.lua

```diff
@@ -0,0 +1,93 @@
+﻿-- This file is subject to copyright - contact swampservers@gmail.com for more information.
+-- INSTALL: CINEMA
+CreateConVar("kleiner_spawncount", "18", FCVAR_ARCHIVE + FCVAR_PROTECTED, "Number of Kleiner NPCs to spawn automatically.", 0, 100)
+KLEINER_NPCS = KLEINER_NPCS or {}
+KLEINER_NPCS_FILTER = KLEINER_NPCS_FILTER or {}
+KLEINER_NPC_TARGETS = KLEINER_NPC_TARGETS or {}
+KLEINER_NPCS_CURRENT_NUMBER = KLEINER_NPCS_CURRENT_NUMBER or 0
+KLEINER_DESIRED_NUMBER = KLEINER_DESIRED_NUMBER or GetConVar("kleiner_spawncount"):GetInt() or 0
+
+function KLEINER_NPC_SETUPTIMER(value)
+    KLEINER_DESIRED_NUMBER = GetConVar("kleiner_spawncount"):GetInt() or 0
+    timer.Destroy("kleiner_spawner")
+
+    if (value > 0) then
```

Comment:
I think this check is unnecessary. Also don't see why you pass `value` to this function when it seems like its always the same value as `KLEINER_DESIRED_NUMBER` which you get inside the function anyway?

Reply:
value is passed from the convar's change callback to update it. This function is called both when the game is initialized, and any point